apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.ts

```diff
@@ -369,4 +363,14 @@ export class RatioAnalysisChartComponent implements OnInit, OnDestroy {
         this.destroyed$.next(true);
         this.destroyed$.complete();
     }
+
+    public translationComplete(event: boolean): void {
+        if(event) {
+            this.rationResponse$.pipe(skipWhile(p => (isNullOrUndefined(p)))).subscribe(p => {
```

Comment:
Add takeUntil, avoid single character name

Reply:
takeUntil is already available in constructor.